src/main/java/step3/Car.java

```diff
@@ -0,0 +1,35 @@
+package step3;
+
+import java.util.Random;
+
+public class Car {
+
+    private static final int MAX_MOVE_LIMIT = 9;
+    private static final int MOVE_START_AT = 4;
+
+    private static final Random random = new Random();
+
+    // 구분
+    private int CAR_NO;
```

Comment:
CAR_NO는 상수가 아니니까 대문자로 쓰지 않아도 될 것 같습니다:)
```suggestion
    private int carNumber;
```

Reply:
네 작업해서 커밋을 하겠습니다.
여기 PR 화면을 보면 Commit Suggetion 이라는 버튼이 보이는데
이것을 클릭하면 리모트의 upstream 에 적용이 되는건가요? 아니면 제 개인 remete origin 에 적용이 되는걸까요...  선뜻 눌렀다가 git 익숙치 않은 상태에서 복잡해질까봐 못 누르겠네요 ㅎㅎ

Reply:
현재 브랜치에 commit 되는 걸로 알고 있습니다:) 현재 리팩토링 하고 커밋하신 것처럼 바로 적용되는 걸로 알고있습니다:)

Reply:
답변 감사합니다. ^^